ha/k8s_setup/ha_setup.py

```diff
@@ -186,6 +186,31 @@ def __init__(self, args):
         """
         super().__init__(args)
 
+    def _get_endpoints(self, num_endpoints_key: str, endpoint_key: str, scheme: str) -> str:
+        """Get endpoints"""
+        ep = None
+        num_endpoints = Conf.get(self._index, num_endpoints_key.format(_DELIM=_DELIM))
+        if num_endpoints is None:
+            Log.error(f"number of endpoints found: {num_endpoints}")
```

Comment:
Could be worded: "Could not get number of endpoints" ?

Reply:
Done. modified a bit.

Reply:
OK

Reply:
Deepak is not able to resolve the comments. I spoke with him and he is fine to close that. Hence closing on his behalf. Thanks!